tests/manage/z_cluster/nodes/test_check_pods_status_after_node_failure.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import pytest
+import random
+
+from ocs_ci.framework.testlib import ManageTest, tier4a, ignore_leftovers
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs.node import (
+    wait_for_nodes_status,
+    get_ocs_nodes,
+    get_nodes_in_statuses,
+    get_node_pods,
+)
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import (
+    wait_for_pods_to_be_running,
+    get_pod_name_by_pattern,
+    get_pod_obj,
+)
+
+log = logging.getLogger(__name__)
+
+
+def get_rook_ceph_pods_not_in_node(node_name):
+    """
+    Get all the rook ceph pods that are not running on the node
+
+    Args:
+        node_name (str): The node name
+
+    Returns:
+        list: List of the rook ceph pod objects that are not running on the node 'node_name'
+
+    """
+    rook_ceph_pod_names_set = set(get_pod_name_by_pattern("rook-ceph-"))
+    node_pods = get_node_pods(node_name)
+    node_pod_names_set = set([p.name for p in node_pods])
+    rook_ceph_pod_names_not_in_node = list(rook_ceph_pod_names_set - node_pod_names_set)
+
+    return [get_pod_obj(pod_name) for pod_name in rook_ceph_pod_names_not_in_node]
+
+
+@ignore_leftovers
+@tier4a
```

Comment:
please create a polarion test case and add the id here

Reply:
I added a Polarion test case https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-2552 and added the id here.